Tests/Units/Cassandra/ORM/SchemaManager.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+namespace CassandraBundle\Tests\Units\Cassandra\ORM;
+
+use mageekguy\atoum\test;
+use CassandraBundle\Cassandra\ORM\SchemaManager as TestedSchemaManager;
+
+class SchemaManager extends test
+{
+	/**
+	 * @dataProvider tableConfigProvider
+	 */
+    public function testCreateTable($tableName, $fields, $primaryKeys, $tableOptions, $expectedCQL)
+    {
+//    	$tableName = 'test';
+//		$fields = [
+//			['columnName' => 'id', 'type' => 'uuid'],
+//			['columnName' => 'name', 'type' => 'text'],
+//		];
+//		$primaryKeys = ['id'];
+//		$tableOptions = ['compactStorage' => true];
+//		$expectedCQL = 'CREATE TABLE test (id uuid,name text,PRIMARY KEY (id)) WITH COMPACT STORAGE;';
```

Comment:
it's better to remove instead of commenting this block

Reply:
Indeed I forgot to remove it